spyderlib/spyder.py

```diff
@@ -2660,6 +2675,133 @@ def show_tour(self, index):
         self.tour.set_tour(index, frames, self)
         self.tour.start_tour()
 
+    # ---- Check for Spyder Updates
+    def _check_updates_ready(self):
+        """Called by WorkerUpdates when ready"""
+        from distutils.version import StrictVersion as Version
+
+        version = __version__
+        latest_release = self.worker_updates.latest_release
+        feedback = self.worker_updates.feedback
+        error_msg = self.worker_updates.error
+        url = 'https://github.com/spyder-ide/spyder/releases'
+
+        # For testing the message boxes when running in bootstrap mode
+        #DEV, version, latest release = None, '2.3.4', '3.0.0'
+
+        if DEV is None:
+            # In case someone installs a release candidate this might fail.
+            try:
+                update_available = Version(latest_release) > Version(version)
+            except ValueError:
+                update_available = False
+        else:
+            update_available = False
+
+        title = _("Spyder updates")
+
+        if error_msg is not None:
+            msg = error_msg
+            QMessageBox.information(self, title, msg)
+        else:
+            if update_available:
+                msg = _('Spyder {0} is available. <br><br>Please use your '
+                        'package manager to update Spyder or go to our '
+                        '<a href="{1}">Releases</a> page to download Spyder '

```

Comment:
You know what would be cool here too? To add a link to the *Installation* page in our documentation. It could something like

*If you don't know to update Spyder, please refer to our [Installation](http://pythonhosted.org/spyder/installation.html) instructions*.